src/index.js

```diff
@@ -35,7 +35,7 @@ export default class Undo {
     );
     observer.setMutationObserver();
 
-    this.setEventListeners();
+    this.setEventListeners(observer.holder);
```

Comment:
@MarioRodriguezS The Observer holder and the **editorjs** holder element are different things, you can't use it in this way :)